lib/vcap_env.rb

```diff
@@ -0,0 +1,13 @@
+require 'json'
+
+# Parses ENV['VCAP_SERVICES'] or similar JSON and sets ENV variables for user-provided:credentials
+module VcapEnv
+  def self.set_env(vcap_json)
+    vcap = JSON.parse(vcap_json)
+    env = vcap["user-provided"].find { |service| service["name"] == "compliance-viewer-env"  }
```

Comment:
Hmm, yeah.  1 "self serve" env variable could make this work, but maybe would be better to put the "sinatra/config.yml" thing back in place, just for this one setting, kinda removes the :egg:.  Or maybe the :chicken:.  Not quite sure.  I'll give that a whirl.